microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -182,66 +188,53 @@ func getServiceURL(service *corev1.Service) string {
 
 func getCustomConfig(step string, svcCfg *map[string]string, yamlFile []byte) (string, error) {
 	var userDefinedCfg interface{}
-	if step == "Embedding" {
+	if step == "Configmap" {
```

Comment:
need to add logic for gaudi cases, this can be address in a follow-up PR though

Reply:
okay, I will add all gaudi cases.

Reply:
done!